ep/build-velox/src/get_velox.sh

```diff
@@ -2,8 +2,10 @@
 
 set -exu
 
-VELOX_REPO=https://github.com/oap-project/velox.git
-VELOX_BRANCH=main
+#VELOX_REPO=https://github.com/oap-project/velox.git
+#VELOX_BRANCH=main
+VELOX_REPO=https://github.com/JkSelf/velox.git
+VELOX_BRANCH=arrow-parquet-write
```

Comment:
can you do a rebase? the dependency patch is merged

Reply:
Yes. Updated.